maintainers/maintainer-list.nix

```diff
@@ -8811,4 +8811,10 @@
     github = "cpcloud";
     githubId = 417981;
   };
+  aciceri = {
```

Comment:
Keep the list of maintainers sorted

Reply:
I added myself at the end because the list isn't sorted, now I put my entry at the beginning in a point where the list is locally sorted. Is this what you meant?